app/js/controllers/foyer/pensionsAlimentaires.js

```diff
@@ -51,31 +49,29 @@ angular.module('ddsApp').controller('FoyerPensionsAlimentairesCtrl', function($s
         $scope.individusVM.push(createIndividuVM(conjoint));
     }
 
-    $scope.hasPensionsAlimentaires = false;
+    $scope.situation.hasPensionsAlimentaires = false;
```

Comment:
What about renaming that variable? `has` is always ambiguous between “receives” and “gives” (there's probably a better verb than “give”).